apiserver/README.md

```diff
@@ -119,744 +157,1841 @@ nodes:
       - containerPort: 31888
         hostPort: 31888
         listenAddress: "0.0.0.0"
-  - role: worker
-  - role: worker
-EOF
-```
-
-1. Deploy the KubeRay APIServer within the same cluster of KubeRay operator
+      - containerPort: 31887 
+        hostPort: 31887 
+        listenAddress: "0.0.0.0"
+    - role: worker
+      image: kindest/node:v1.23.17@sha256:59c989ff8a517a93127d4a536e7014d28e235fb3529d9fba91b3951d461edfdb
+    - role: worker
+      image: kindest/node:v1.23.17@sha256:59c989ff8a517a93127d4a536e7014d28e235fb3529d9fba91b3951d461edfdb
+    EOF
+    ```
 
-```bash
-helm repo add kuberay https://ray-project.github.io/kuberay-helm/
-helm -n ray-system install kuberay-apiserver kuberay/kuberay-apiserver
-```
+2. Deploy the KubeRay APIServer within the same cluster of KubeRay operator
 
-2. The APIServer expose service using `NodePort` by default. You can test access by your host and port, the default port is set to `31888`.
+    ```sh
+    helm repo add kuberay https://ray-project.github.io/kuberay-helm/
+    helm -n ray-system install kuberay-apiserver kuberay/kuberay-apiserver -n ray-system --create-namespace
+    ```
 
-```
-curl localhost:31888
-{"code":5, "message":"Not Found"}
-```
+3. The APIServer expose service using `NodePort` by default. You can test access by your host and port, the default port is set to `31888`. The examples below assume a kind (localhost) deployment. If Kuberay API server is deployed on another type of cluster, you'll need to adjust the hostname to match your environment.
```

Comment:
You can use both NodePort and Ingress

Reply:
Cool, but I think we should takle the documentation for how to create the ingress as separate PR.